components/actionbar/index.css

```diff
@@ -85,7 +85,8 @@ governing permissions and limitations under the License.
   block-size: var(--spectrum-actionbar-height);
   min-inline-size: var(--spectrum-actionbar-min-width);
   max-inline-size: var(--spectrum-actionbar-max-width);
-  padding: 0 var(--spectrum-actionbar-padding-right) 0 var(--spectrum-actionbar-padding-left);
+  padding-inline-start: var(--spectrum-actionbar-padding-left);
+  padding-inline-end: var(--spectrum-actionbar-padding-right);
```

Comment:
Should these CSS Custom Properties also leverage logical naming devices, e.g. `--spectrum-actionbar-padding-start/end`?

Reply:
They come out of DNA with `left`/`right` currently, but it might be something we should do in the future @NateBaldwinDesign